.github/workflows/unittests.yml

```diff
@@ -140,8 +142,11 @@ jobs:
         if: steps.ch_version.outputs.triggered == 'true'
       - name: Run Gluten + ClickHouse Backend unit tests with Spark 3.2
         run: |
+          pushd /tmp
+          git clone --depth 1 --branch v3.2.2 https://github.com/apache/spark.git
+          popd
           export MAVEN_OPTS="-Xmx5g -XX:ReservedCodeCacheSize=1g"
-          mvn clean install -Pbackends-clickhouse -Pspark-3.2 -Pspark-ut -Dtpcds.data.path=/home/changchen/tpcds-sf1-data -Dclickhouse.lib.path=/tmp/libch.so
+          mvn clean install -Pbackends-clickhouse -Pspark-3.2 -Pspark-ut -Dtpcds.data.path=/home/changchen/tpcds-sf1-data -Dclickhouse.lib.path=/tmp/libch.so -DargLine="-Dspark.test.home=/tmp/spark"
```

Comment:
Hi @zzcclp, a correct Spark home is needed for this suite to use some testing files. Could you help check if this change is proper for CH backend? If there is already a Spark-3.2 on the test node, we can also directly use that.

Reply:
Please download the Spark-3.2 for it now, there is not a Spark-3.2 on the test node.

Reply:
Got it, thanks.